site/src/util/mdx-utils.tsx

```diff
@@ -194,9 +204,11 @@ export const getPage = (params: {
     sections: headings.reduce<SiteMapPageSection[]>((prev, currentHeading) => {
       const newSection = {
         title: getVisibleText(currentHeading),
-        anchor: slugify(getFullText(currentHeading), {
-          lower: true,
-        }),
+        anchor:
+          currentHeading.anchor ||
```

Comment:
It feels like it would be better and less user-error prone to get rid of the need to add custom `anchor` to FAQ entries, and instead just strip out any HTML from FAQ titles here when setting the anchor. But I appreciate this is a pre-existing problem

Reply:
Done in d6e89df82491d6b0f722760175b4f8083a20afab. Let me know if something is missing

Reply:
Not missing, but there are still provisions for an `anchor` property on headings that no longer exists, including here